pkg/crc/network/nameservers.go

```diff
@@ -74,12 +74,17 @@ func HasNameserverConfiguredLocally(nameserver NameServer) (bool, error) {
 
 func GetResolvValuesFromHost() (*ResolvFileValues, error) {
 	// TODO: we need to add runtime OS in case of windows.
-	out, err := ioutil.ReadFile("/etc/resolv.conf")
+	fInfo, err := crcos.GetFilePath("/etc/resolv.conf")
```

Comment:
I must be missing something, as I far as I can tell, ioutil.ReadFile already follows symlinks:
```
package main

import "bytes"
import "io/ioutil"
import "log"
import "os"


        
func main() {
	const content = "test file content"
        var byte_content = []byte(content)

	err := ioutil.WriteFile("/tmp/realfile", byte_content, 0644)
	if err != nil {
		log.Fatal(err)
	}
	err = os.Symlink("/tmp/realfile", "/tmp/symlink")
	if err != nil {
		log.Fatal(err)
	}
			        
	out, err := ioutil.ReadFile("/tmp/symlink")
	if err != nil {
		log.Fatal(err)
	}
	if !bytes.Equal(out, byte_content) {
		log.Fatal("symlink content does not match expected content")
	}
	log.Println("Symlink test worked as expected")
}
```

Reply:
@cfergeau, you are not missing anything, `ioutil.ReadFile` able to read the file which is even symlink but I added if in future we required some symlink path, also I hope this is not causing any issue.